src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(Color color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: triangle, area: " + calculateArea() + "sq. units, firstLeg: "
```

Comment:
Use only English in messages/code. The message printed should be in English to make the code universal and consistent.